src/Manifest/Manifest.csproj

```diff
@@ -13,11 +13,19 @@
   <Import Project="..\..\build\NetFrameworkRelease.targets" />
 
   <ItemGroup>
-    <EmbeddedResource Include="$(MSBuildProjectDirectory)\..\bld\ReleaseInfo.json" />
+    <EmbeddedResource Include="$(TargetDir)ReleaseInfo.json" LogicalName="AccessibilityInsights.Manifest.ReleaseInfo.json" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Octokit" Version="5.0.0" />
```

Comment:
Is this only retrieved when doing a release build/building the manifest?

Reply:
Even though this project only _builds_ in release, the `nuget restore` step will always fetch this, even in debug. This is visible at [this step](https://dev.azure.com/accessibility-insights/Accessibility%20Insights/_build/results?buildId=44571&view=logs&j=54c8d87f-fb30-55c8-e653-392071771ae4&t=c36c8f7e-19fe-59c9-6025-4f2e0cb2c38b&l=653) of the PR build. I'll add a comment about keeping the version in sync with the PowerShell script, though